src/domain/Cards/Card/Card.tsx

```diff
@@ -154,6 +157,14 @@ class Card extends React.PureComponent<ICardProps, ICardState> {
     return undefined
   }
 
+  private get target (): string | undefined {
```

Comment:
was gonna say that this seems unneeded because `this.props.target` will be string | undefined anyways, but it looks like the other props do this already so whatever